projects/plugins/jetpack/_inc/client/components/jetpack-notices/user-license-activation.jsx

```diff
@@ -89,11 +97,18 @@ const UserLicenseActivationNotice = props => {
 				showDismiss={ true }
 				onDismissClick={ onNoticeDismiss }
 				text={ createInterpolateElement(
-					__( 'You have an inactive product. <a>Activate now</a>', 'jetpack' ),
+					__( 'You have an available product license key. <activateLink>Activate it now</activateLink> or <purchasesLink>view all your purchases</purchasesLink>', 'jetpack' ),
 					{
-						a: createElement( 'a', {
+						activateLink: createElement( 'a', {
 							href: USER_LICENSE_ACTIVATION_ROUTE,
-							onClick: trackClick,
+							onClick: trackLicenseActivationClick,
+						} ),
+						purchasesLink: createElement( ExternalLink, {
+                            className: 'jp-license-activation-notice__external-link',
```

Comment:
Could you replace spaces by tabs? :)

```suggestion
							className: 'jp-license-activation-notice__external-link',
```

Reply:
Sure. Sorry about this. :)